lib/gis/testsuite/test_gis_lib_getl.py

```diff
@@ -0,0 +1,76 @@
+"""Test of gis library line reading functions
+
+@author Vaclav Petras
+"""
+
+import ctypes
+import pathlib
+import platform
+
+import grass.lib.gis as libgis
+from grass.gunittest.case import TestCase
+from grass.gunittest.main import test
+
+
+class TestNewlinesWithGetlFunctions(TestCase):
+    """Test C functions G_getl() and G_getl2() from gis library"""
+
+    @classmethod
+    def setUpClass(cls):
+        cls.libc = ctypes.cdll.LoadLibrary(ctypes.util.find_library("c"))
```

Comment:
```suggestion
        cls.libc = ctypes.cdll.LoadLibrary(ctypes.util.find_library("c"))
        FILE_ptr = ctypes.POINTER(libgis.FILE)
        cls.libc.fopen.restype = FILE_ptr
        cls.libc.fopen.argtypes = [ctypes.c_char_p, ctypes.c_char_p]
```

Reply:
Thank you. Is this something which will work across platforms or should I put that for macOS only? In other words, should I apply this as is?

Reply:
This should work cross platform, I'm surprised it worked at all without.

Reply:
I see. Maybe just on Linux. ctypes is always more difficult then I expect.

Reply:
The ubuntu based test also failed with similar result as the mac.